x-pack/plugins/cases/public/components/files/utils.test.tsx

```diff
@@ -47,3 +48,37 @@ describe('parseMimeType', () => {
     expect(parseMimeType('foo/bar')).toBe('Foo');
   });
 });
+
+describe('isValidFileExternalReferenceMetadata', () => {
+  it('should return false for empty objects', () => {
+    expect(isValidFileExternalReferenceMetadata({})).toBeFalsy();
+  });
+
+  it('should return false if the files property is missing', () => {
+    expect(isValidFileExternalReferenceMetadata({ foo: 'bar' })).toBeFalsy();
+  });
+
+  it('should return false if the files property is not an array', () => {
+    expect(isValidFileExternalReferenceMetadata({ files: 'bar' })).toBeFalsy();
+  });
+
+  it('should return false if file.length !== 1', () => {
```

Comment:
nit: I don't think that the function returns false because of the `file.length !== 1`. I think it returns false because it is not an array of valid metadata. The check in the `isValidFileExternalReferenceMetadata` is `externalReferenceMetadata?.files?.length >= 1`. Maybe we can delete the test or we can change it to check that it validates an array of two valid metadata.

Reply:
You're right. This is a leftover from the previous implementation of `isValidFileExternalReferenceMetadata`.